src/sqlfluff/core/rules/base.py

```diff
@@ -455,15 +456,15 @@ def _validate_config_options(self, config, rule=None):
         Config options can also be checked for a specific rule e.g L010.
         """
         rule_config = config.get_section("rules")
-
+        from icecream import ic

```

Comment:
What's this line? While there _is_ a package called `icecream`, it's not listed in the requirements files.